llama_stack/cli/stack/run.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, subparsers: argparse._SubParsersAction):
 
     def _add_arguments(self):
         self.parser.add_argument(
-            "config",
+            "--config",
```

Comment:
I’m not sure why we’re changing from a positional argument to a flag, but I do prefer the flag. However, this seems outside the scope of the unit tests. Can you clarify?

Reply:
Thanks, let’s revisit this in another PR.